### ISSUE TITLE
CLI : transmission-create command create a corrupted .torrent file

### ISSUE THREAD

**Issue by H0R1Z0N** [transmission/transmission#6609]
### What is the issue?

I have been trying to create an automate script to upload a torrent, it seem that transmission-create in the terminal have issue with the integrity of the output .torrent. Adding the .torrent to the transmission-gtk or transmission-cli fail to scan the folder source. If i do the same with the GUI on the GTK version, it's properly detected in both gtk and cli client. I have the latest cli version on Arch from the extra repository, 4.0.5-1. I have tried to simplify my directory name to be sure it was not a bug related to me using the bash value $(date +... but it was not the case. When i check the corrupted generated file list of the transmission-create .torrent in the GUI, it list all my file properly, so i assume it might be hashing issue. I have not specifie any specific piece size either.

### Which application of Transmission?

None

### Which version of Transmission?

transmission-create 4.0.5 (a6fe2a64aa)

**Comment by tearfur** [comment]
> Adding the .torrent to the transmission-gtk or transmission-cli fail to scan the folder source.

Could you elaborate on what you mean by this? Any screenshots you can show of the torrent and file status?

**Comment by H0R1Z0N** [comment]
Yes, so i am trying to automate a script for my server, i have attempted multiple way doing the syntax in case it was it but it did not change anything.

```
transmission-create -t http://tracker.url/example "Folder used" -o "Folder used.torrent"
```

Then, tried to load it with the following command for seeding :
```
transmission-cli -v -w "Folder used" "Folder used.torrent"
```

The verify skip and don't find the file, in case i had the wrong comamnd, i did it also with transmission-gtk and it also fail to detect the folder's file, even if the preview of the file in the folder in the propriety display properly. I had tried to see if the fact i used some bash date command to generate the file name was the culprit, it was not the case. Generating the torrent using transmission-gtk work flawlessly, but since i was to make it happen using a SystemD Timer for automated backup, it's not what i desired. I have tried to see if eliminating the anynomyse flag was an issue or not. It's for use on an internal server has a backup solution mirror for our indie game project.

**Comment by tearfur** [comment]
I think the tool you want is `transmission-remote` instead of `transmission-cli`.

`transmission-cli` is a legacy standalone Transmission client that allows you to download/seed a single torrent.

**Comment by H0R1Z0N** [comment]
> I think the tool you want is `transmission-remote` instead of `transmission-cli`.
> 
> `transmission-cli` is a legacy standalone Transmission client that allows you to download/seed a single torrent.

It still failed with the transmission-gtk too with the transmission-create generated torrent.

**Comment by tearfur** [comment]
Did you enabled RPC in the GTK client? It is in the `Remote` tab of the preference dialogue. Also in case you didn't notice, the syntax of `transmission-remote` is different from `transmission-cli`.

**Comment by H0R1Z0N** [comment]
No, that not the issue, i principally want to generate a torrent in the terminal using transmission-create, but no client detect the file properly once i use the .torrent i generated using transmission-create. In the end, i don't mind what ever's the software to use for the client, it's for generating a working .torrent from the terminal for using with SystemD timer.

**Comment by tearfur** [comment]
> No, that not the issue

Right, sorry, I understand now. Let me take a look.

Meanwhile, if you have sample files that can reproduce this behaviour with `transmission-create`, please attach them here.

**Comment by ckerr** [comment]
This ticket hasn't been touched in a couple off years and I don't know how to proceed with it as-is.

@H0R1Z0N if this is repeatable in 4.1.0, could you please write up a recipe that the maintainers can follow to reproduce the issue?

**Comment by H0R1Z0N** [comment]
I have no more need for it since i had to do it manually for a while until
i stopped doing so. Been too long since i wanted to automate my step.

**Comment by ckerr** [comment]
Closing stale ticket based on age and lack of actionable info.
